src/app/drive/components/MoveItemsDialog/MoveItemsDialog.tsx

```diff
@@ -163,11 +165,9 @@ const MoveItemsDialog = (props: MoveItemsDialogProps): JSX.Element => {
         await restoreItemsFromTrash(itemsToMove, destinationFolderId, translate as TFunction, props.isTrash);
       }
 
-      props.onItemsMoved?.();
```

Comment:
Don't remove this, this will cause it not to execute what is being passed through this prop.